proto/erc20.proto

```diff
@@ -287,6 +301,31 @@ message RewardsDenyListUpdated {
   uint64 ordinal = 3;
 }
 
+message SubgraphPublished1 {
+  string id = 1;
+  bytes graph_account = 2;
+  string subgraph_number = 3;
+  bytes subgraph_deployment_id = 4;
+  bytes version_metadata = 5;
+  uint64 ordinal = 6;
+}
+
+message SubgraphPublished2 {
+  string id = 1;
+  string subgraph_id = 2;
+  bytes subgraph_deployment_id = 3;
+  string reserve_ratio = 4;
+  uint64 ordinal = 5;
+}
+
+message SubgraphVersionUpdated {
+  string id = 1;
+  string subgraph_id = 2;
+  bytes subgraph_deployment_id = 3;
+  bytes version_metadata = 4;
+  uint64 ordinal = 5;
+}
```

Comment:
Can't remember now if there was another `SubgraphVersionUpdated` that had graph account + subgraph number instead of subgraph id, but I think there was 🤔

Reply:
Nevermind, I got it confused with `SubgraphMetadataUpdated` which existed both in the "graph account + subgraph number" format and the subgraph id format.